src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import exception.InvalidTaskException;
+import exception.NoDescriptionException;
+import exception.WrongDescriptionException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
```

Comment:
A possible suggestion would be to add header comments for the tester classes too.